package.json

```
@@ -1,55 +1,23 @@
 {
-  "name": "measure-repository-service",
+  "name": "measure-repository",
   "version": "0.0.1",
-  "description": "Prototype implementation of a FHIR Measure Repository Service",
-  "license": "Apache-2.0",
-  "main": "dist/index.js",
+  "workspaces": [
+    "frontend",
+    "backend"
+  ],
   "scripts": {
-    "build": "tsc -p ./tsconfig.build.json",
-    "build:watch": "tsc -p ./tsconfig.build.json -w",
-    "check": "npm run test && npm run lint && npm run prettier",
-    "db:delete": "ts-node ./scripts/dbSetup.ts delete",
-    "db:reset": "ts-node ./scripts/dbSetup.ts reset",
-    "db:setup": "ts-node ./scripts/dbSetup.ts create",
-    "db:loadBundle": "ts-node ./scripts/dbSetup.ts loadBundle",
-    "lint": "eslint \"./src/**/*.{js,ts}\"",
-    "lint:fix": "eslint \"./src/**/*.{js,ts}\" --fix",
-    "prettier": "prettier --check \"./src/**/*.{js,ts}\"",
-    "prettier:fix": "prettier --write \"src/**/*.{js,ts}\"",
-    "start": "ts-node ./src/index.ts",
-    "start:watch": "nodemon ./src/index.ts",
-    "start:clean": "npm run db:reset && npm run start",
-    "test": "jest",
-    "test:watch": "jest --watch"
+    "build:all": "npm run build --workspaces",
+    "check:all": "npm run test --workspace=backend && npm run lint --workspace=backend && npm run prettier --workspace=backend && npm run lint --workspace=frontend && npm run prettier --workspace=backend",
+    "lint:all": "npm run lint --workspaces",
+    "lint:all-fix": "npm run lint:fix --workspaces",
+    "prettier:all": "npm run prettier --workspaces",
+    "prettier:all-fix": "npm run prettier:fix --workspaces",
```

not sure if this is better or convention, idk. Doesn't really matter

```suggestion
    "lint:all:fix": "npm run lint:fix --workspaces",
    "prettier:all": "npm run prettier --workspaces",
    "prettier:all:fix": "npm run prettier:fix --workspaces",
```